ci/coredns-configmap.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: kube-system
+  name: coredns
+data:
+  Corefile: |
+    .:53 {
+        errors
+        health {
+           lameduck 5s
+        }
+        ready
+        kubernetes cluster.local in-addr.arpa ip6.arpa {
+           pods insecure
+           ttl 30
+        }
+        prometheus :9153
+        forward . [{{ k8s_antrea_gw_ip }}]:5353 {
+           max_concurrent 1000
+        }
+        cache 30
+        reload
+        loadbalance
+    }
+    vmware.com:53 {
```

Comment:
> Read coredns document. From my understanding, this section means that coredns returns a "non-existent domain" error code for all DNS client requesting vmware.com. Is my understanding correct? I'm not sure how it solves the issue of IPv6-only cluster requesting a v4 DNS server.

This is added only for IPV6 only case, so here for an IPv6 only cluster, this would just enforce to return NXDOMAIN and it means that the requested domain name does not exist in the DNS namespace.  This config will just tell dns queries wrt vmware.com to not to go to the upstream resolver !

Reply:
@KMAnju-2021 @edwardbadboy 
Further in IPv6-only clusters, CoreDNS tries to talk to the dockerd upstream DNS server via IPv4 and fails and this part needs to be handled with NXDOMAIN ! 
Usually dns resolution is required for external request into the cluster practically !
However for the  local cluster dns resolution,  resolving dns works as expected and gets handled locally , see below

[INFO] [fd00:10:244:1::f]:60038 - 11223 "AAAA IN kubernetes.default.svc.cluster.local.cluster.local. udp 79 false 4096" NXDOMAIN qr,aa,rd 161 0.000119972s
[INFO] [fd00:10:244:1::f]:42868 - 16658 "AAAA IN kubernetes.default.svc.cluster.local.eng.vmware.com. udp 80 false 4096" - - 0 0.000096571s
[ERROR] plugin/errors: 2 kubernetes.default.svc.cluster.local.eng.vmware.com. AAAA: dial udp 172.18.0.1:53: connect: network is unreachable